label_audit.py

```diff
@@ -18,31 +20,45 @@ def check_js(paths, strings_dict):
 
     for path in paths:
 
-        short_path = path.replace(os.path.join("force-app", "main", "default", ""),"")
+        if (
+            os.path.basename(path) == "moment.js"
+            or os.path.basename(path) == "chart.js"
+            or "__tests__" in path
```

Comment:
I was talking with Scott about another blacklist and I wonder if we ignore anything that starts with __?

Reply:
Per slack conversation, we will wait until this is needed.